magpie/utils.py

```diff
@@ -19,6 +21,7 @@
 import six
 import sys
 import os
+
 if TYPE_CHECKING:
     from magpie.definitions.typedefs import (  # noqa: F401
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'str'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650807)